pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -359,17 +359,19 @@ func (r *MigPlan) UpdateRegistryDeployment(storage *MigStorage, deployment *apps
 	deployment.Spec = appsv1.DeploymentSpec{
 		Replicas: pointer.Int32Ptr(1),
 		Selector: metav1.SetAsLabelSelector(map[string]string{
-			"app":        name,
-			"deployment": name,
-			"migplan":    string(r.UID),
+			"app":                  name,
+			"deployment":           name,
+			"migplan":              string(r.UID),
+			MigrationRegistryLabel: string(r.UID),
```

Comment:
Shouldn't label constant be in the api too?

Reply:
It's here: https://github.com/konveyor/mig-controller/blob/master/pkg/apis/migration/v1alpha1/migplan_types.go#L236

But actually, now that I think about it, we should probably remove it from the controller and just reference `migapi.MigrationRegistryLabel` instead of definining it twice.